<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PySync</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Audio-File</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="audioFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectAudioFile">
        <property name="text">
         <string>Select File...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Subtitle-File</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="subtitleFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectSubtitleFile">
        <property name="text">
         <string>Select File...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Output-File</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="outputFile">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectOutputFile">
        <property name="text">
         <string>Select File...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Model-Size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="model">
      <property name="currentIndex">
       <number>6</number>
      </property>
      <item>
       <property name="text">
        <string>tiny</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tiny.en</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>base</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>base.en</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>small.en</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>medium.en</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>large</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="language">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="doSync">
        <property name="text">
         <string>Synchronize</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
